chore: Style and SEO updates (#3775)
hashicorp/terraform-cdk: website/docs/cdktf/api-reference/provider.mdx

---
page_title: CDKTF API Reference provider bindings
description: >-

>   Learn how CDKTF uses provider bindings to make your CDKTF application run with different Terraform providers.

Author: ```suggestion
  Learn how CDK for Terraform uses provider bindings to make your CDKTF application run with different Terraform providers.
```

---

> page_title: CDKTF API Reference provider bindings

Reviewer: Can you make the H1 match?

description: >-

>   The CDKTF provider bindings are used to make your CDKTF application run with different Terraform providers

Reviewer: ```suggestion
 Learn how CDKTF uses provider bindings to make your CDKTF application run with different Terraform providers.
```
